util/types/errors.go

```diff
@@ -37,35 +39,46 @@ var (
 	ErrWrongFieldSpec = terror.ClassTypes.New(codeWrongFieldSpec, "Wrong Field Spec")
 	// ErrBadNumber is return when parsing an invalid binary decimal number.
 	ErrBadNumber = terror.ClassTypes.New(codeBadNumber, "Bad Number")
+	// ErrCastSignedOverflow is returned when positive out-of-range integer, and convert to it's negative complement
```

Comment:
1. s/ ErrCastSignedOverflow/ ErrCast**As**SignedOverflow be better?
2. add `.` at the end